doc/sphinx/source/esmvalcore/preprocessor.inc

```diff
@@ -198,100 +392,106 @@ matches the named regions against the requested string.
 See also :func:`esmvalcore.preprocessor.extract_named_regions`.
 
 
-Volume manipulation
-===================
+Volume operations
+=================
+
 The _volume.py module contains the following preprocessor functions:
 
-* extract_volume: Extract a specific depth range from a cube.
-* volume_statistics: Calculate the volume-weighted average.
-* depth_integration: Integrate over the depth dimension.
-* extract_transect: Extract data along a line of constant latitude or longitude.
-* extract_trajectory: Extract data along a specified trajectory.
+1. extract_volume: extract a specific depth range from a cube.
```

Comment:
Make this links to the relevant subsections below

Reply:
Done.